Coffee/Coffee/Application/SceneDelegate.swift

```diff
@@ -16,7 +16,23 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
         // Use this method to optionally configure and attach the UIWindow `window` to the provided UIWindowScene `scene`.
         // If using a storyboard, the `window` property will automatically be initialized and attached to the scene.
         // This delegate does not imply the connecting scene or session are new (see `application:configurationForConnectingSceneSession` instead).
-        guard let _ = (scene as? UIWindowScene) else { return }
+        guard let windowScene = (scene as? UIWindowScene) else { return }
+        self.window = UIWindow(windowScene: windowScene)
+
+        // Set Onboarding storyboard as initial storyboard
+        let storyboard = UIStoryboard(name: "Onboarding", bundle: nil)
```

Comment:
Can create enum for Storyboards?
I think it can be easy to use and prevent from wrong typo issue.
How do you think?